ir/shared/src/main/scala/org/scalajs/ir/Traversers.scala

```diff
@@ -245,4 +245,18 @@ object Traversers {
     }
   }
 
+  /** Traverser that only traverses the local scope.
+   *
+   *  In practice, this means stopping at `Closure` boundaries: their
+   *  `captureValues` are traversed, but not their other members.
+   */
+  abstract class LocalScopeTraverser extends Traverser {
+    override def traverse(tree: Tree): Unit = tree match {
+      case Closure(_, _, _, _, _, captureValues) =>
+        captureValues.foreach(traverse(_))
```

Comment:
`case closure: Closure => closure.captureValues.foreach(traverse(_))`?

Reply:
I prefer the current form: In case we change the parameters to the `Closure` tree, the compiler will point out points where we need to make a decision on whether to use or ignore the new parameters.

If we use the suggested form, that is not the case.